src/logion/controllers/locrequest.controller.ts

```diff
@@ -43,17 +57,32 @@ export class LocRequestController extends ApiController {
     @HttpPost('')
     @Async()
     async createLocRequest(createLocRequestView: CreateLocRequestView): Promise<LocRequestView> {
-        return this.fake(createLocRequestView, "REQUESTED");
+        this.authenticationService.authenticatedUserIs(this.request, createLocRequestView.requesterAddress);
+        const description: LocRequestDescription = {
+            requesterAddress: requireDefined(createLocRequestView.requesterAddress),
+            ownerAddress: requireDefined(createLocRequestView.ownerAddress),
+            description: requireDefined(createLocRequestView.description),
+            createdOn: moment().toISOString()
```

Comment:
This will return a "zoned" datetime (which may be fine but it is important to know :-) ).